frontend/src/components/UsernamePasswordListTile.vue

```diff
@@ -15,50 +15,66 @@ limitations under the License.
 -->
 
 <template>
-  <div v-show="(!!username || !!email) && !!password">
-    <v-list-item v-if="username">
+  <div v-show="((!!username || !!email) && !!password) || showNotAvailablePlaceholder">
+    <template v-if="(!!username || !!email) && !!password">
```

Comment:
The code became to complex for html template from my point of view. We should create a computed property `showCredentials` or something like this.

```suggestion
  <div v-show="showCredentials || showNotAvailablePlaceholder">
    <template v-if="showCredentials">
```

Reply:
done